app/views/custom/proposals/_geozone.html.erb

```diff
@@ -0,0 +1,5 @@
+<ul id="geozone" class="no-bullet geozone">
+  <li class="inline-block">
```

Comment:
It's a bit confusing to me that we define the `<ul>` to be `inline-block` in the CSS code but for the `<li>` we use the HTML. How about moving this style to the CSS file?